dash-renderer/src/components/core/DocumentTitle.react.js

```diff
@@ -5,14 +5,16 @@ import PropTypes from 'prop-types';
 class DocumentTitle extends Component {
     constructor(props) {
         super(props);
+        const {update_title} = props.config;
         this.state = {
             initialTitle: document.title,
+            update_title,
         };
     }
 
     UNSAFE_componentWillReceiveProps(props) {
-        if (props.isLoading) {
-            document.title = 'Updating...';
```

Comment:
@Marc-Andre-Rivet as we were discussing the other day - the renderer no longer holds this default value at all. That means when the R and Julia implementations update to include this functionality in their renderer, they will *need* to add this option to the back end at the same time or the default behavior will change to no title changes ever.